features/marketdatacomponent.feature

```diff
@@ -52,6 +52,8 @@ Feature: Any user should be able see Market data component
         And I should see "Change (%)" within "homePage.headerRow"
         And I verify words are uppercase for "homePage.allRowsCompanyName"
         Then I verify "homePage.listingsDirectoryLink" link location is "/listings_directory/stock"
-
         
-        
\ No newline at end of file
+    Scenario: 05 - Verify Company Name check for company page title
```

Comment:
this test appears to be a duplication of the scenario 4. 
Is there any vaild reason why you need this test to be added ?